cmd/exporter/main.go

```diff
@@ -43,135 +45,141 @@ var registeredResultMetrics []string
 
 // Declare the fixed chaos metrics. Dynamic (testStatus) metrics are defined in metrics()
 var (
-    experimentsTotal = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-        Namespace: "c",
-        Subsystem: "engine",
-        Name:      "experiment_count",
-        Help:      "Total number of experiments executed by the chaos engine",
-    },
-    []string{"app_uid", "engine_name"},
-    )
-
-    passedExperiments = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-        Namespace: "c",
-        Subsystem: "engine",
-        Name:      "passed_experiments",
-        Help:      "Total number of passed experiments",
-    },
-    []string{"app_uid", "engine_name"},
-    )
-
-    failedExperiments = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-        Namespace: "c",
-        Subsystem: "engine",
-        Name:      "failed_experiments",
-        Help:      "Total number of failed experiments",
-    },
-    []string{"app_uid", "engine_name"},
-    )
-
+	experimentsTotal = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "c",
+		Subsystem: "engine",
+		Name:      "experiment_count",
+		Help:      "Total number of experiments executed by the chaos engine",
+	},
+		[]string{"app_uid", "engine_name", "kube_version"},
```

Comment:
can we explain what these array of string imply?
can we have this defined separately with a well understood variable name?

Reply:
We could use separate variable name such as env_variables, or static_info.
Would that do the part?

Reply:
this array of string is needed as part of declaration of var of type `prometheus.NewGaugeVec`. It is common for all the metrics we are using and hence could be reused, but it may depend upon the metric.